shared/build.gradle.kts

```diff
@@ -9,6 +9,9 @@ plugins {
     kotlin(Plugin.KOTLIN_SERIALIZATION)
     id(Plugin.NATIVE_COROUTINES).version(Version.NATIVE_COROUTINES_KOTLIN)
     id(Plugin.BUILD_KONFIG)
+    id(Plugin.MOCKMP).version(Version.MOCKMP)
+    id(Plugin.KSP).version(Version.KSP)
+    id(Plugin.KOVER)
```

Comment:
it is noticed that sometimes you add the `version` here and sometimes don't. Shouldn't we need to add the `versions` for all the plugins? 🤔

Reply:
These version are defined in [root/build.gradle](https://github.com/blyscuit/kmm-swiftUI-IC/blob/feature/%239-backend-login/build.gradle.kts) so we can't redefine them here. The Plugins with version are those that the root build.gradle does not need them 😄